### ISSUE TITLE
Small jei compatibility error.

### ISSUE THREAD

**Issue by wasdspace** [infernalexp/Mining-Master#26]
### Describe the Bug

When generating the world:

```
[Render thread/ERROR] [mezz.jei.recipes.RecipeManagerInternal/]: Found a broken recipe: Mining Master miningmaster:forging/spirit_garnet_pickaxe class org.infernalstudios.miningmaster.recipes.ForgingRecipe
Outputs:
net.minecraft.item.ItemStack: [[1 air minecraft:air]]
Inputs:
net.minecraft.item.ItemStack: [[1 spirit_garnet miningmaster:spirit_garnet], [1 spirit_garnet miningmaster:spirit_garnet], [1 spirit_garnet miningmaster:spirit_garnet], [1 diamond_pickaxe minecraft:diamond_pickaxe nbt:{Damage:0}]]

java.lang.IllegalArgumentException: ItemStack value must not be empty. 1 air minecraft:air
	at mezz.jei.util.ErrorUtil.checkNotEmpty(ErrorUtil.java:206) ~[jei:7.7.1.152]
	at mezz.jei.plugins.vanilla.ingredients.item.ItemStackHelper.getUniqueId(ItemStackHelper.java:88) ~[jei:7.7.1.152]
	at mezz.jei.plugins.vanilla.ingredients.item.ItemStackHelper.getUniqueId(ItemStackHelper.java:28) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeMap.addRecipe(RecipeMap.java:94) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeMap.addRecipe(RecipeMap.java:75) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeManagerInternal.addRecipe(RecipeManagerInternal.java:125) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeManagerInternal.addRecipeTyped(RecipeManagerInternal.java:101) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeManagerInternal.addRecipes(RecipeManagerInternal.java:94) ~[jei:7.7.1.152]
	at mezz.jei.recipes.RecipeManagerInternal.<init>(RecipeManagerInternal.java:86) ~[jei:7.7.1.152]
	at mezz.jei.load.PluginLoader.getRecipeManager(PluginLoader.java:145) ~[jei:7.7.1.152]
	at mezz.jei.startup.JeiStarter.start(JeiStarter.java:74) ~[jei:7.7.1.152]
	at mezz.jei.startup.ClientLifecycleHandler.startJEI(ClientLifecycleHandler.java:147) ~[jei:7.7.1.152]
	at mezz.jei.startup.ClientLifecycleHandler.setupJEI(ClientLifecycleHandler.java:140) ~[jei:7.7.1.152]
	at mezz.jei.startup.ClientLifecycleHandler.lambda$new$3(ClientLifecycleHandler.java:106) ~[jei:7.7.1.152]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-4.0.0.jar:?]
	at net.minecraftforge.client.ForgeHooksClient.onRecipesUpdated(ForgeHooksClient.java:772) ~[forge:?]
	at net.minecraft.client.network.play.ClientPlayNetHandler.func_199525_a(ClientPlayNetHandler.java:1314) ~[?:?]
	at net.minecraft.network.play.server.SUpdateRecipesPacket.func_148833_a(SourceFile:27) ~[?:?]
	at net.minecraft.network.play.server.SUpdateRecipesPacket.func_148833_a(SourceFile:15) ~[?:?]
	at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?]
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?]
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(SourceFile:103) ~[?:?]
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:948) ~[?:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) ~[?:?]
	at net.minecraft.client.main.Main.main(Main.java:184) ~[1.16.5-Forge_36.2.35.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:37) ~[forge-1.16.5-36.2.35.jar:36.2]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.1.3.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.1.3.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.1.3.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.1.3.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.1.3.jar:?]
```

### Reproduction Steps

1.
2.
3.
...

### Expected Result

Although it may not affect the play, it may also be a small mistake.

### Operating System

Windows 11

### Mod Version

2.1.0

### Minecraft Version

1.16.5

### Forge Version

36.2.35

### Other Mods

jei: 7.7.1.152

### Additional Context

Mod Version: 3.0.4        ------      lastest version on curseforge, not 2.1.0

**Comment by Dariensg** [comment]
Ah, thank you! I found and fixed this issue while porting to 1.18 actually and forgot to go back and fix it for 1.16. I will try to get this fixed soon! Thank you for reminding me!

**Comment by Dariensg** [comment]
Fixed by fa0c8b0296f0fba0880285924a7da472ca4ee708. Will release new version with fix soon
